### ISSUE TITLE
@@odata.id causing IDs Should Be Derived From ResourceIDs

### ISSUE THREAD

**Issue by LeonB87** [Azure/arm-ttk#232]
I have the following code in my ARM-Template that does a HTTP call to the Graph API

```code
{
	"HTTP_Add_Groupmember": {
		"runAfter": {
			"HTTP": [
				"Succeeded"
			]
		},
		"type": "Http",
		"inputs": {
			"authentication": {
				"audience": "https://graph.microsoft.com",
				"clientId": "@string(body('HTTP_Get_Client_Id').value)",
				"secret": "@string(body('HTTP_Get_Client_Secret').value)",
				"tenant": "@string(body('HTTP_Get_Tenant_Id').value)",
				"type": "ActiveDirectoryOAuth"
			},
			"body": {
				"@@odata.id": "https://graph.microsoft.com/v1.0/directoryObjects/@{json(decodeBase64(triggerBody()?['ContentData'])).userid}"
			},
			"headers": {
				"Content-type": "application/json"
			},
			"method": "POST",
			"uri": "@{concat('https://graph.microsoft.com/v1.0/groups/',string(body('HTTP').value[0].id),'/members/$ref')}"
		}
	}
}
```

This causes ARM-TTK to fail

![image](https://user-images.githubusercontent.com/10503724/99379782-67350e80-28c9-11eb-81a6-86b0b92636a9.png)

Adding "@@Odata.Id" to the exception list "fixes" this issue.

![image](https://user-images.githubusercontent.com/10503724/99379882-8633a080-28c9-11eb-9bb1-3a240f2d2996.png)

Creating Fork to suggest fix.

**Comment by LeonB87** [comment]
Hi @bmoore-msft ; could you kindly look at the PR for this issue? Regards, Léon

**Comment by LeonB87** [comment]
This is fixed in PR [#234 ](https://github.com/Azure/arm-ttk/pull/234 )
